.circleci/config.yml

```diff
@@ -2,7 +2,7 @@ version: 2
 jobs:
   build:
     docker:
-      - image: circleci/php:5.6
+      - image: circleci/php:7.1
```

Comment:
Your requirement is PHP 5.6 or later, so please allow me to suggest a rollback for this new PHP version unless you're dropping support for PHP 5.6 or PHP 7.0, to be consistent with the minimum required PHP version.

Reply:
As mentioned, this is the separate PR for the PHP upgrade: https://github.com/intercom/intercom-php/pull/271

We're working on the last details before merging it. I have a local version of this PR rebased with the PHP 7 branch + the change to the readme you suggested. I'll push it here as soon as the other PR is merged 😃

Reply:
Added a suggestion for the README file, but the PR makes sense to me 👍